Gopkg.lock

```diff
@@ -78,24 +78,14 @@
 [[projects]]
   branch = "master"
   name = "github.com/etcd-io/gofail"
-  packages = [
-    ".",
-    "code",
-    "runtime"
-  ]
+  packages = ["runtime"]
   revision = "51ce9a71510a58bad5ae66ddd278ef28762a1550"
 
 [[projects]]
   name = "github.com/go-sql-driver/mysql"
   packages = ["."]
   revision = "3955978caca48c1658a4bb7a9c6a0f084e326af3"
 
-[[projects]]
-  name = "github.com/gogo/protobuf"
```

Comment:
do we not use gogo now? @tiancaiamao

Reply:
I think update pd and kvproto should not affect those packages.

Reply:
I didn't know what's wrong but each time I run 
> dep ensure -update github.com/pingcap/pd
the gogo package will be replaced with google/protobuffer

Reply:
@liukun4515 FYI